src/core/prompts/model_prompts/claude4.ts

```diff
@@ -122,16 +122,21 @@ Usage:
 <file_pattern>file pattern here (optional)</file_pattern>
 </search_files>
 
-## list_files
-Description: Request to list files and directories within the specified directory. If recursive is true, it will list all files and directories recursively. If recursive is false or not provided, it will only list the top-level contents. Do not use this tool to confirm the existence of files you may have created, as the user will let you know if the files were created successfully or not.
-Parameters:
-- path: (required) The path of the directory to list contents for (relative to the current working directory ${cwd.toPosix()})
-- recursive: (optional) Whether to list files recursively. Use true for recursive listing, false or omit for top-level only.
+## LS: Lists directory contents
```

Comment:
If you were to rebase on latest main and then push again that would be the best thing for testing this.